cli/cmd/internal/client/target.go

```diff
@@ -1 +1,52 @@
 package client
+
+import (
+	"code.cloudfoundry.org/quarks-utils/pkg/cmd"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/suse/carrier/cli/paas"
+)
+
+var ()
+
+// CmdTarget implements the carrier target command
+var CmdTarget = &cobra.Command{
+	Use:   "target [org]",
+	Short: "Targets an organization in Carrier.",
+	Args:  cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, cleanup, err := paas.NewCarrierClient(cmd.Flags(), nil)
+		defer func() {
+			if cleanup != nil {
+				cleanup()
+			}
+		}()
+
+		if err != nil {
+			return errors.Wrap(err, "error initializing cli")
+		}
+
+		org := ""
+		if len(args) > 0 {
+			org = args[0]
+		}
+
+		err = client.Target(org)
```

Comment:
Neither nor in the called function is `org` validated to exist.

UI nit: In my role as the user deploying an app I am unable to tell if the `target` command has an actual effect, as `push` the app does not tell which org the app is pushed into.

Only as a developer can I see that the code of the `Push` backend function was updated to take the org into account.

I am ok with fixing both these things after the PR is merged.

Reply:
Will be done in #68